docs/openapi.yaml

```diff
@@ -224,7 +224,7 @@ paths:
                       - type: string
                     description: >
                       Unique ID of the created Course.  Exact type/format will depend on your implementation but will likely be either an integer or a string.
-                    example: "123"
+                    example: 123
```

Comment:
Why?

Reply:
In our implementation, all IDs are integers.